dockers/docker-config-engine-buster/Dockerfile.j2

```diff
@@ -1,5 +1,5 @@
 {% from "dockers/dockerfile-macros.j2" import install_debian_packages, install_python_wheels, copy_files %}
-FROM docker-base-buster
+FROM {{ docker_config_engine_buster_load_image }}
```

Comment:
Is it possible to define a variable contains the info that we can easily know which image it bases on?
For instance, {{ docker_base_buster_image }}
All the images can be exposed as variables before building dockers.
docker_base_buster_image=docker-base-buster-$(user)

Reply:
Done